apeer_dev_kit/_core.py

```diff
@@ -18,8 +22,8 @@ def _get_inputs(self):
             self.inputs = Namespace(**self._input_json)
             return self.inputs
         except KeyError:
-            print(
-                "Environment variable WFE_INPUT_JSON not found. Please add WFE_INPUT_JSON as an environment variale. Example of WFE_INPUT_JSON: {\"WFE_output_params_file\":\"wfe_module_params_1_1.json\", \"red\":0, \"green\":0.5, \"blue\":0}")
+            raise KeyError(
+                'Environment variable WFE_INPUT_JSON not found. Please add WFE_INPUT_JSON as an environment variale to get inputs')
```

Comment:
Shouldn't this error message refer to WFE_output_params_file not found? --because that what could hapepen in the try.

Reply:
yups, copy paste error